packages/wallet/test/hardware/ledger/LedgerKeyAgent.test.ts

```diff
@@ -747,6 +748,26 @@ describe('LedgerKeyAgent', () => {
                 )
               ).toBe(true);
             });
+
+            // TODO: Enable once we have confirmation from Ledger team that it is supported
+            it.skip('can sign with drep key', async () => {
```

Comment:
If it's failing then it seems that it's not supported? 🤔

Reply:
Yes, I suspect it's not supported and that is why it's failing, but I will need confirmation from the Ledger team. If that is the case, I'll remove the test altogether

Reply:
It works. I have updated the implementation https://github.com/input-output-hk/cardano-js-sdk/compare/f6e83827681b3ff08e8e43bb02eb59fe23591a4f..99a5930a9243df29a0eae1daeae4ed29c29a98ca